content/influxdb/cloud-dedicated/query-data/execute-queries/troubleshoot.md

```diff
@@ -0,0 +1,274 @@
+---
+title: Understand and troubleshoot Flight responses
+description: >
+  Understand responses and troubleshoot errors encountered when querying {{% cloud-name %}} with Flight+gRPC and Arrow Flight clients.
```

Comment:
The shortcode won’t work in frontmatter. Hugo just reads it as a string literal. I could make it so it does work, but it's a little hacky.

Reply:
😊 I knew that